File: nest/src/model/category/category.entity.ts
import { BaseModel } from 'shared/base.model';
import { Column, Entity } from 'typeorm';

@Entity()
export class Category extends BaseModel<Category> {
  @Column({ type: 'nvarchar', length: 120 })
  name: string;

  @Column({ type: 'nvarchar', length: 255 })
  description: string;

  @Column({ type: 'int' })
  showOrder: number;
}

File: quarkus/dishmanagement/src/main/java/com/devonfw/application/dishmanagement/domain/model/CategoryEntity.java
package com.devonfw.application.dishmanagement.domain.model;

import javax.persistence.Entity;
import javax.persistence.Table;

import com.devonfw.application.general.domain.model.ApplicationPersistenceEntity;

import lombok.Getter;
import lombok.Setter;

/**
 * The {@link com.devonfw.application.mtsj.general.dataaccess.api.ApplicationPersistenceEntity persistent entity} for
 * {@link CategoryEntity}.
 */
@Getter
@Setter
@Entity
@Table(name = "Category")
public class CategoryEntity extends ApplicationPersistenceEntity {

  private String name;

  private String description;

  private int showOrder;

}
